modules/5_install/install.tf

```diff
@@ -211,8 +211,50 @@ resource "null_resource" "configure_public_vip" {
     }
 }
 
+
+resource "null_resource" "setup_snat" {
+    count       = var.setup_snat ? var.bastion_count : 0
+    depends_on  = [null_resource.config]
+
+    connection {
+        type        = "ssh"
+        user        = var.rhel_username
+        host        = var.bastion_public_ip[count.index]
+        private_key = var.private_key
+        agent       = var.ssh_agent
+        timeout     = "15m"
+    }
+
+    provisioner "remote-exec" {
+        inline = [<<EOF
+
+echo "Configuring SNAT (experimental)..."
+
+PRIVATE_INTERFACE=$(ip r | grep "${var.cidr} dev" | awk '{print $3}')
+PUBLIC_INTERFACE=$(ip r | grep "${var.public_cidr} dev" | awk '{print $3}')
+
+firewall-cmd --zone=public --add-masquerade --permanent
+# Masquerade will enable ip forwarding automatically
+firewall-cmd --reload
+
+#sudo yum install -y iptables
+iptables -t nat -A POSTROUTING -o $PUBLIC_INTERFACE -j MASQUERADE
+iptables -A FORWARD -i $PRIVATE_INTERFACE -j ACCEPT
+iptables -A FORWARD -o $PRIVATE_INTERFACE -j ACCEPT
```

Comment:
afaik, if using firewall-cmd, we don't need the above iptables rules

Reply:
Removed the iptables commands.